build.sh

```diff
@@ -14,13 +14,14 @@ for target in libambit movescount openambit openambit-cli
 do
     cd ${SOURCE_LOCATION}
     echo "------building $target------"
-    mkdir -p ${target}-build
-    cd ${target}-build
-    cmake "$@" ../src/${target}
-    make -j${CORES}
-    if [ "${DO_INSTALL-0}" == "1" ]; then
-      echo "------installing $target------"
-      sudo make install
+    mkdir -p $target-build
+    cd $target-build
+    cmake "$@" ../src/$target
+    make -j$CORES
+    if [ "$DO_INSTALL" == "1" ]; then
+    echo "------installing $target------"
+    sudo make install
+    sudo ldconfig
```

Comment:
There are some unrelated newline/blank changes, maybe you can revert them?

Reply:
ok, I will revert them